README.md

```diff
@@ -164,6 +166,20 @@ API names and versions are generated by parsing the `info` section of the defini
 
 If an error occurs while uploading any definition, the build will fail and subsequent definitions will not be uploaded.
 
+### SCM integration provisioning considerations
+
+* Supported SCM's include: `GITHUB`, `BITBUCKET`
+
+* Care should be taken when specifying SCM parameters. Validation does not take prior to making the request to SwaggerHub and issues can arise due to incorrectly configured integrations
```

Comment:
Validation does not take **place** prior

Reply:
Updated 👍